### PR TITLE
update yaml docs

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

**Comment by pwright** [comment]
addresses #4

File: modules/declarative/pages/index.adoc [hunk 1]
```diff
@@ -53,4 +53,93 @@ skupper-site-controller-84694bdbb5-n8slb   1/1     Running   0          75s
...
+  service-sync: "true"
+kind: ConfigMap
+metadata:
+  name: skupper-site
+----
+
+. Apply the YAML file to your cluster:
++
+----
+kubectl apply -f ~/east-site.yml
```
Comment (ajssmith) [inline modules/declarative/pages/index.adoc:92]: Above, the file is named 'my-site.yaml'

**Review by ajssmith — APPROVED** [review]
This lgtm. 

With changes coming to the cli, the data/labels for the site .yaml will also be updated soon.

File: modules/declarative/pages/index.adoc (hunk 1 above)
Comment (pwright) [reply modules/declarative/pages/index.adoc]: resolved

**Pull request merged**